packages/app/src/components/AmountInput.tsx

```diff
@@ -61,6 +61,7 @@ const AmountInput: React.FC<AmountInputProps> = ({
               className="text-blue hover:bg-outset hover:cursor-pointer py-1 px-2 rounded-md"
               onClick={() => {
                 if (balance) {
+                  setValid(true);
```

Comment:
I think instead of using this, change setAmount to updateAmount

Reply:
Done